### PR TITLE
Charts + Hub

### PR DESCRIPTION
**Pull request by thedeath154**
Almost done, all that is left is the proper json and convert from scrollable charts to all on the same page

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -81,9 +81,9 @@
     "esModuleInterop": true,                             /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables 'allowSyntheticDefaultImports' for type compatibility. */
     // "preserveSymlinks": true,                         /* Disable resolving symlinks to their realpath. This correlates to the same flag in node. */
     "forceConsistentCasingInFileNames": true,            /* Ensure that casing is correct in imports. */
-
-    /* Type Checking */
-    "strict": true,                                      /* Enable all strict type-checking options. */
```
Comment (daniel-mita) [inline tsconfig.json:86]: Don't remove this, if there is a typescript error you need to solve it, not remove the strict type checking

**Review by daniel-mita — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (daniel-mita) [inline yarn.lock:1]: Why do you have both yarn.lock and package-lock? Decide on using either yarn or npm and remove the other

File: src/components/Charts/Chart.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import data from "/Users/SCP/Workshop/meeting-rooms-booking/src/Charts.json";
+interface iChart {
+    data: { name: string; uv: number }[];
+}
+
+const Chart = ({ data }: iChart) => {
+    return (
+        <Box sx={{ display: "flex", flexDirection: "column" }}>
+            <Box sx={{ display: "flex", alignItems: "center", mt: 1, ml: 1 }}>
+                <IconButton component={Link} to="http://localhost:3000/hub">
```
Comment (robert-szalkai) [inline src/components/Charts/Chart.tsx:16]: you should NOT use absolute paths. Use relative paths `to="/hub"`

File: src/components/Charts/Chart.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                </BarChart>
+            </Box>
+        </Box>
+    );
+};
+const AllCharts = () => {
+    return (
+        <Box sx={{ display: "flex", flexDirection: "column" }}>
+            <Box sx={{ display: "flex", alignItems: "center", mt: 1, ml: 1 }}>
+                <IconButton component={Link} to="http://localhost:3000/hub">
```
Comment (robert-szalkai) [inline src/components/Charts/Chart.tsx:53]: same here

File: src/components/Hub/Hub.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    return (
+        <div
+            style={{
+                display: "flex",
+                justifyContent: "center",
+                alignItems: "center",
+                height: "100vh",
+            }}
+        >
+            <a href="http://localhost:3000/selection">
```
Comment (robert-szalkai) [inline src/components/Hub/Hub.tsx:13]: same here

**Review by robert-szalkai — CHANGES_REQUESTED** [review]

File: meetingRoomData.json [hunk 5]
```diff
@@ -46,25 +134,5 @@
...
-
-
-
-   
-
-
-
-
-
-
```
Comment (Liubzzz) [inline meetingRoomData.json:58]: unnecessary spaces

File: meetingRoomData.json [hunk 6]
```diff
@@ -46,25 +134,5 @@
...
-
-
-
-
```
Comment (Liubzzz) [inline meetingRoomData.json:64]: same here

File: src/components/Charts/Chart.tsx [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+                            left: 20,
+                            bottom: 5,
+                        }}
+                    >
+                        <CartesianGrid strokeDasharray="3 3" />
+                        <XAxis dataKey="name" />
+                        <YAxis />
+                        <Bar
+                            dataKey="value"
+                            fill="#8884d8"
```
Comment (Liubzzz) [inline src/components/Charts/Chart.tsx:152]: use constants here, for filling colors in charts

**Pull request closed without merging**